listenbrainz_spark/hdfs_connection.py

```diff
@@ -6,3 +6,7 @@ def init_hdfs(namenode_uri, user='root'):
     global client
     client = hdfs.InsecureClient(namenode_uri, user='root')
 
+def init_test_hdfs(namenode_uri, user='root'):
```

Comment:
why is this needed? Shouldn't the normal hdfs connection function work? We use the same hostname in tests.